spec/system/accessibility_spec.rb

```diff
@@ -1,13 +1,113 @@
 require 'rails_helper'
 
 RSpec.describe 'Accessibility' do
-  include_context 'with a logged in user'
-  before { driven_by(:headless_chrome) }
-
-  describe 'Start page' do
-    it 'has no AXE-detectable accessibility issues' do
-      visit '/'
-      expect(page).to be_axe_clean
-    end
+  include_context 'with a logged in admin user'
+  include_context 'with an existing user'
+  include_context 'with an existing application'
+
+  let(:accessibility_standards) { [:wcag2a, :wcag2aa] }
+
+  before do
+    driven_by(:headless_chrome)
+    visit '/'
+  end
+
+  it 'viewing assigned applications page has no axe detectible accessibility issues' do
+    expect(page).to be_axe_clean.according_to accessibility_standards
+  end
+
```

Comment:
In my [previous PR](https://github.com/ministryofjustice/laa-review-criminal-legal-aid/pull/181), I was using a shared example to reduce the repetitiveness of the code. However, this made it difficult to test routes with their url helper names e.g. `open_crime_applications_path` vs. hardcoding the url as a shared example can only be used in describe and content blocks whereas a url helper can only be used in if blocks.

Reply:
👍

Reply:
I think this is fine - it may be a little repretitive but its easy to read and understand which sometimes is more important imo.

If it becomes too long we can all ways refactor it 👍